enhancements/installer/install_ca_management.md

```diff
@@ -0,0 +1,161 @@
+
+### Implementation Details/Notes/Constraints
+
+The main constraint is that nodes need this CA information before the node-ca-daemon can be started (nodes need the CA to pull images, the node-ca-daemon is an image), so while the node-ca-daemon is the first class mechanism for getting CAs from the image config api to the nodes, it cannot be relied upon for purposes of bootstrapping the cluster.  Therefore the existing mechanism must continue to function at least through bootstrapping the cluster.  After that point, the CAs should come from the image config api.
+
+
+### Risks and Mitigations
+
+The main risk is that we get the bootstrapping/configuring logic/ordering incorrect and end up with clusters(and in particular nodes) that do not have the right CAs to trust the mirror registry.
+
+Otherwise this EP largely amounts to having the installer automate steps that we already tell the cluster admin do today, so there is not substantial risk.
+
+## Design Details
+
+### Open Questions
+
+1. What is the current install->mco->node CA flow and does it continue reconciling values post install?  Does it compete with the node-ca-daemon?
+
+2. Where does the installer get the mirror registry hostname from so it knows whether or not to populate the image config resource?  (Presumably it has this information since it populates the image content source policy)
+
+
+### Test Plan
+
+Perform a disconnected install with a mirror registry that uses a custom CA.  Ensure that post install:
+- the image config resource + associated CA configmap are properly configured
+- nodes can pull images from the mirror
+- imagestreams can import images from the mirror
+- builds can pull images from the mirror
+- the internal registry can perform image pullthrough against the mirror
+
+Perform a standard (non-disconnected) install but supply an additionalTrustBundle to the installer.  Ensure that post install:
+- the image config resource was not populated by the installer
+
+
+### Graduation Criteria
+
+This should move directly to GA state once implemented.  
+
+### Upgrade / Downgrade Strategy
+
+N/A
+
+### Version Skew Strategy
+
+N/A
+
+## Drawbacks
+
+Makes the installer responsible for additional cluster configuration logic.
+The additionalTrustBundle field is ambiguous, this design further overloads it, increasing the ambiguity about exactly what the values provided there will be used for and when.
+
+
+## Alternatives
+
+Introduce new fields in the install config where CAs can be explicitly provided for separate purposes and deprecate the existing additionalTrustBundle field.  So far we have identified two types of CAs that are potentially needed:
+
+1. CAs for the proxy (including MITM proxies)
+2. CAs for the mirror registry
```

Comment:
A third one - CAs for private registries that are not used as mirrors.

Reply:
as that is not needed to get the cluster up and running with consistent config/behavior, it's not something we need to collect at install time.

the admin can add CAs for those registries post install.

The goal here is to not have different CAs on the nodes vs what other components use, not to provide an install-mechanism to generally configure the cluster.

Reply:
i've added a non-goal to clarify/address this